pkg/api/graph/interfaces.go

```diff
@@ -38,7 +38,7 @@ const (
 type Markers []Marker
 
 // MarkerScanner is a function for analyzing a graph and finding interesting things in it
-type MarkerScanner func(g Graph, f Namer) []Marker
+type MarkerScanner func(g Graph, f Namer, chaseRefs bool) []Marker
```

Comment:
The marking process isn't mutating the graph, it just needs to know if we are chasing references so it will not suggest --chase-refs.